src/django/api/tests/test_facility_merge.py

```diff
@@ -213,6 +213,12 @@ def setUp(self):
             self.merge_endpoint, self.facility_1.id, self.facility_2.id
         )
 
+    def set_up_updated_at_field(self):
```

Comment:
You are declaring `set_up_updated_at_field()` multiple times, pls move it to `src/django/api/helpers/helpers.py` the and import from there.

Reply:
Also, update branch to the current state.

Reply:
I use the exact test context in these in these methods. I'm not sure I can move it into a helper.